contrib/epee/include/net/http_server_impl_base.h

```diff
@@ -60,8 +61,16 @@ namespace epee
       const std::string& bind_ipv6_address = "::", bool use_ipv6 = false, bool require_ipv4 = true,
       std::vector<std::string> access_control_origins = std::vector<std::string>(),
       boost::optional<net_utils::http::login> user = boost::none,
-      net_utils::ssl_options_t ssl_options = net_utils::ssl_support_t::e_ssl_support_autodetect)
+      net_utils::ssl_options_t ssl_options = net_utils::ssl_support_t::e_ssl_support_autodetect,
+      const std::size_t max_public_ip_connections = 3,
+      const std::size_t max_private_ip_connections = 25,
+      const std::size_t max_connections = 100,
+      const std::size_t response_soft_limit = 25 * 1024 * 1024)
```

Comment:
We should put this `25 * 1024 * 1024` value, which shows up 3 times in the patch, into a named constant

Reply:
I was avoiding this, because I had no idea where to place such a thing. Looks like good old `cryptonote_config.h` will work, even with epee.